test/functionalTest/cases/1289_exceed_mongo_key_limit/exceed_mongo_key_limit.test

```diff
@@ -0,0 +1,86 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'd suggest to create a "parallel" .test (e.g. exceed_mongo_key_limit.test
) to cover this funcionality also in /v1. For example:

01. POST /v1/updateContext (using in the payload an entity id/type violating the constraint), you have examples of how to do updateContext in existing .test
02. GET /v1/contextEntities/Room1